pkg/controller/controller.go

```diff
@@ -97,16 +97,25 @@ func (c *Controller) initOperatorConfig() {
 		c.logger.Infoln("no ConfigMap specified. Loading default values")
 	}
 
-	// env var takes priority over the same param from the operator ConfigMap
-	watchedNamespace := os.Getenv("WATCHED_NAMESPACE")
-	if watchedNamespace != "" {
-		c.logger.Infof("Watch the %q namespace specified in the env variable WATCHED_NAMESPACE\n", watchedNamespace)
-		configMapData["watched_namespace"] = watchedNamespace
+	// by default, the operator listens to all namespaces
+	if configMapData["watched_namespace"] == "" {
+		c.logger.Infoln("The operator config map specifies no namespace to watch. Falling back to watching all namespaces.")
```

Comment:
Note that if the key is missing or empty in the ConfigMap but the environment variable is specified, this will still be printed to the log.  While this arguably correct (as the log message does refer explicitly to the config map), it could be confusing to see this message say it will fall back to watching all namespaces, and right below that say it's actually going to do what the environment variable specifies.